resources/lang/en/message.php

```diff
@@ -0,0 +1,16 @@
+<?php
+return [
+    'list' => 'List Customers',
+    'no' => 'No',
+    'name' => 'Name',
+    'age' => 'Age',
+    'address' => 'Address',
+    'email' => 'Email',
+    'phone' => 'Phone',
+    'options' => 'Options',
+    'create' => 'Add new Customer',
+    'submit' => 'Submit',
+    'back' => "Go Back",
+    'whoops' => "Whoops",
+    'wrong' => 'There were some problems with your input.'
```

Comment:
```suggestion
    'wrong' => 'There were some problems with your input.',
```
mảng có nhiều phần tử thì phần tử cuối thêm dấu "," nhé :D